File: Contest_Solutions/Code-e/02_Spring_2021_Contests/Contest_15/C++/Efficient_Cashier_Easy_Version/main.cpp
#include <bits/stdc++.h>
using namespace std;
typedef long long ll;

int main() {
    int t;
    cin>>t;
    ll a[4] = {25, 10, 5, 1};
    while(t--){
        ll x;
        cin>>x;
        ll ans = 0;
        for(int i = 0; i < 4; i++){
            ans += x/a[i];
            x %= a[i];
        }
        cout<<ans<<endl;
    }

    return 0;
}

File: Contest_Solutions/Code-e/02_Spring_2021_Contests/Contest_15/Java/Efficient_Cashier_Easy_Version/main.java
import java.util.*;
public class main{
	public static void main( String[] args ){
		Scanner cin = new Scanner(System.in);
		int t = cin.nextInt();
		while(t > 0){
		    long n = cin.nextLong();
    		long ans = 0;
    		int[] coins = {25, 10, 5, 1};
    		for(int c: coins){
    		    ans += n / c;
    		    n %= c;
    		}
    		System.out.println(ans);
		    t--;
		}
	}
}